elisp/proto/generate.el

```diff
@@ -14,6 +14,8 @@
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
 
+;; Version: 0.1.0
```

Comment:
consider moving these into a separate PR since they are somewhat unrelated

Reply:
Done https://github.com/phst/rules_elisp/pull/367

Reply:
note that I have rebased this PR's commit onto that PR's commit for my convenience.  I will rebase+push after that PR is submitted.